src/main/java/hudson/plugins/brakeman/BrakemanResult.java

```diff
@@ -89,6 +116,118 @@ public String getDisplayName() {
+    }
+
+    /**
+     * Updates the build status, i.e. sets this plug-in result status field to
+     * the corresponding {@link Result}. Additionally, the {@link Result} of the
+     * build that owns this instance of {@link BuildResult} will be also
+     * changed.
+     *
+     * This has been overridden to add a different evaluate strategy that
+     * evaluates thresholds against non-ignored warnings/annotations.
+     *
+     * @param thresholds
+     *            the failure thresholds
+     * @param useDeltaValues
+     *            the use delta values when computing the differences
+     * @param canComputeNew
+     *            determines whether new warnings should be computed (with
+     *            respect to baseline)
+     * @param logger
+     *            the logger
+     * @param url
+     *            the URL of the results
+     */
+    // CHECKSTYLE:OFF
+    @SuppressWarnings("hiding")
+    @Override
+    public void evaluateStatus(final Thresholds thresholds, final boolean useDeltaValues, final boolean canComputeNew,
+                               final PluginLogger logger, final String url) {
+        // CHECKSTYLE:ON
+        if(isSuccessful()) {
+            this.thresholds = thresholds;
+            this.useDeltaValues = useDeltaValues;
+
+            BuildResultEvaluator resultEvaluator = new BuildResultEvaluator(url);
+            Result buildResult;
+            StringBuilder messages = new StringBuilder();
+            Set<FileAnnotation> annotations = getNonIgnoredAnnotations();
+            if (getHistory().isEmpty() || !canComputeNew) {
+                logger.log("Ignore new warnings since this is the first valid build");
+                buildResult = resultEvaluator.evaluateBuildResult(messages, thresholds, annotations);
+            } else if (useDeltaValues) {
+                buildResult = resultEvaluator.evaluateBuildResult(messages, thresholds, annotations,
+                        getDelta(), getHighDelta(), getNormalDelta(), getLowDelta());
+            } else {
+                buildResult = resultEvaluator.evaluateBuildResult(messages, thresholds,
+                        annotations, getNewWarnings());
+            }
+            String myReason = messages.toString();
+            setReason(myReason);
+
+            setResult(buildResult);
+
+            logger.log(String.format("%s %s - %s", Messages.Brakeman_ResultAction_Status(), buildResult.color.getDescription(), getReason()));
+        }
+    }
+
+    public Set<FileAnnotation> getIgnoredAnnotations() {
+        Set<FileAnnotation> myAnnotations = getAnnotations();
+        Iterator<FileAnnotation> itr = myAnnotations.iterator();
+        Set<FileAnnotation> annotations = new HashSet<FileAnnotation>();
+        while(itr.hasNext()) {
+            FileAnnotation a = itr.next();
+            if(a.getCategory() == "Ignored") {
+                annotations.add(a);
+            }
+        }
+
+        return annotations;
```

Comment:
I agree with this. I want to be protective though because I don't know if the tabs file can have any categories besides general so I might keep this the same for the time being